css/style.css

```diff
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;1,300&family=Roboto&family=Roboto+Condensed:wght@300&display=swap");
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 20px;
+  min-height: 70px;
+}
+
+.nav-menu {
+  display: flex;
+  align-items: center;
+  gap: 4rem;
+  margin-left: 2rem;
+  margin-right: 3rem;
+}
+
+.nav-branding {
+  font-size: 2rem;
+  color: #6070ff;
+}
+
+.nav-link {
+  transition: all 0.5s ease;
+}
+
+.hambuger {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: #6070ff;
+}
+
+@media all and (max-width: 680px) {
+  .hambuger {
+    display: block;
+  }
+
+  .nav-menu {
+    position: fixed;
+    right: 100%;
+    gap: 0;
+    top: 70px;
+    width: 100%;
+    text-align: center;
+    flex-direction: column;
+    margin: 0;
+    transition: 0.5s;
+  }
+
+  .nav-item {
+    padding: 15px 0;
+  }
+
+  .nav-menu.active {
+    right: 0;
+  }
+}
+
+/* Headline */
+.headline {
+  min-height: 80vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 10px 30px;
+}
+
+.headline-bg {
+  background-image: url(/images/Header-bg2x.png);
```

Comment:
- [ ] Please make sure you have the right background image, you used the background image for the desktop version, not for mobile, you can find it in Figma as well :grin: :grin: 
Your design
![Screenshot from 2022-07-01 16-16-12](https://user-images.githubusercontent.com/74501523/176912042-2f658a00-d83a-4cb4-acc1-77f1e305bc85.png)
 Figma design

![Screenshot from 2022-07-01 16-20-18](https://user-images.githubusercontent.com/74501523/176912590-96046f81-de3d-4eea-b596-a4d486a42a53.png)

Reply:
Already about to exhaust my reviews. Will send in for the last review the necessary changes.